css/style.css

```diff
@@ -2229,47 +2005,6 @@ the home page, intro pitch and your-code-here layed out vertically */
         content: "\f0c9"; /* bars */
         font-family: FontAwesome;
     }
-
-    .org_list a img
-    {
-        padding: 1em 2em;
-    }
-
-    .call_to_donate
-    {
-        padding: 40px 80px;
-    }
-}
-
-/* Narrow layout stage 3: no more two column boxes */
-@media only screen and (max-width: 40em)
-{
-    .boxes .row, .boxes .row .item
```

Comment:
These are also used by the foundation page.

Reply:
Argh. Good catch! Thanks!